app/app/api/public-cloud/products/[licencePlate]/route.ts

```diff
@@ -24,8 +26,14 @@ export const PUT = createApiHandler({
 
 export const DELETE = createApiHandler({
   roles: [GlobalRole.User],
-  validations: { pathParams: deletePathParamSchema },
-})(async ({ pathParams, session }) => {
-  const response = await deleteOp({ session, pathParams });
+  validations: {
+    pathParams: deletePathParamSchema,
+    body: z.object({
+      requestComment: commentSchema,
```

Comment:
it is DELETE method, which body is not available; please create another route for `archiving`